src/components/LoanCards/Buttons/ActionButton.vue

```diff
@@ -68,6 +81,13 @@ export default {
 			default: false,
 		},
 	},
+	data() {
+		return {
+			addToBasketRedirectExperimentShown: false,
+			hasFreeCredits: false,
+			userPrefContinueBrowsing: false,
```

Comment:
I don't think we should store/check this. We should just be using whether they have something already in their basket or if they're using promo credits.

Reply:
@emuvente I initially thought this as well, the intended behavior is: 
1. Be part of experiment, have no promo credits, get redirected to basket. 
2. Click "Continue Browsing"
3. Add another loan, or don't, check out. 
4. Come back next week, add something to basket, they should NOT be redirected to basket under any circumstance, because they previously clicked "Continue Browsing"

Reply:
Where is that step 4 requirement? I don't see that in the ticket anywhere.

Reply:
I verified with Brandon that "_didn’t previously click “continue browsing loans”_" means, we should save it in localStorage and should last as long as possible